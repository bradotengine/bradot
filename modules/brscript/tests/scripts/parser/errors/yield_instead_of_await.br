#BRTEST_PARSER_ERROR

signal event

func test():
	yield("event")
