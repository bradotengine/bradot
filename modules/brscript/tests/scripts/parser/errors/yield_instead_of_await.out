BRTEST_PARSER_ERROR
"yield" was removed in Bradot 4. Use "await" instead.
